nbautoexport/nbautoexport.py

```diff
@@ -44,10 +46,18 @@ def main(
         help="Show nbautoexport version.",
     ),
 ):
-    """Exports Jupyter notebooks to various file formats (.py, .html, and more) upon save,
-    automatically.
+    """Automatically export Jupyter notebooks to various file formats (.py, .html, and more) upon
```

Comment:
Ah just noticing that `typer` very nicely adjusts the width of these docstrings when they are printed out in the command line. Now the docstrings can be written with column width of 100 in the source code, but printed out to column width of 80 for the 0.1% of ppl who use ancient monitors.

Reply:
Yeah, this actually annoyed me recently for a different project where I wanted to put some ASCII art in my help string but `typer` totally destroyed it with formatting. 😂 😢